app/assets/javascripts/directives.coffee

```diff
@@ -0,0 +1,33 @@
+angular.module('movesApp').directive 'updateModelOnEnterKeyPressed', ->
```

Comment:
Can't seem to find where these directives are used; are we planning on using them later?